src/sentry/api/endpoints/project_group_index.py

```diff
@@ -471,18 +471,13 @@ def put(self, request, project):
             groups_to_delete = []
 
             for group in group_list:
-                with transaction.atomic():
-                    try:
+                try:
+                    with transaction.atomic():
                         tombstone = GroupTombstone.objects.create(
                             previous_group_id=group.id,
                             actor_id=acting_user.id if acting_user else None,
                             **{name: getattr(group, name) for name in TOMBSTONE_FIELDS_FROM_GROUP}
                         )
-                    except IntegrityError:
-                        # in this case, a tombstone has already been created
-                        # for a group, so no hash updates are necessary
-                        pass
-                    else:
                         groups_to_delete.append(group)
```

Comment:
why did you take this out of the else block? is it to make sure the `GroupHash` objects get updated also and if not, it gets rolled back?